typing/printpat.mli

```diff
@@ -15,8 +15,13 @@
 
 
 
-val pretty_const : Asttypes.constant -> string
-val top_pretty : Format.formatter -> Typedtree.pattern -> unit
-val pretty_pat : Typedtree.pattern -> unit
-val pretty_line : Format.formatter -> Typedtree.pattern list -> unit
-val pretty_matrix : Format.formatter -> Typedtree.pattern list list -> unit
+val pretty_const
+    : Asttypes.constant -> string
```

Comment:
Kind of picky, but this is the first time I see this format in a `mli`  (i.e. starting a line with `:`).

Reply:
It's used in other modules, see [asmcomp/cmmgen.mli](https://github.com/ocaml/ocaml/blob/60f5817/asmcomp/cmmgen.mli#L18-L22) for example.

Reply:
It isn't an example of the style you use here, but of the _Jane Street style_.
For instance, in `misc.mli`:
```
    val is_prefix
       : equal:('a -> 'a -> bool)
      -> 'a list
      -> of_:'a list
      -> bool
```
That is, write one argument per line, the first one starting with `:`, and others with `->`.

What you did here is write the whole type on one line.
I did grep for something similar:
```
grep "^ *:[^(]*->" */*.mli */*/*.mli
ocamltest/actions_helpers.mli:  : bool -> string -> string -> out_channel -> Environments.t
middle_end/flambda/inline_and_simplify_aux.mli:  : round:int -> Inlining_cost.Threshold.t
```
I.e. only 2 instances in the whole distribution. I see no advantage to this style.

Reply:
I don't see any advantage to this style (I don't remember why I wrote things this way, except that I had to break lines to avoid a 80-column overrun), but I don't see any disadvantage either. Why bother discussing it at all? Surely there are more interesting things to be discussing on this work or the other on `type_pat`.

Reply:
Good question. It stroke me as unusual, and it is apparently. Do you plan to turn it into a fad?
I have no deep objection, but I find it less readable (I admit it's purely instinctive, maybe influenced by punctuation rules in natural language).